packages/cli/src/components/PosthogScript.tsx

```diff
@@ -1,4 +1,4 @@
-import { POSTHOG_API_KEY } from "../util/postHog/client";
+import { POSTHOG_API_KEY } from "../util/postHog/posthogApiKey";
```

Comment:
really would have thought tree shaking would work here 🤷

Reply:
Yeah, it is incredible (-y bad) that it doesn't!